File: 252-meeting-rooms/252-meeting-rooms.java
class Solution {
    public boolean canAttendMeetings(int[][] inter) {
        Arrays.sort(inter, (a,b)-> Integer.compare(a[0],b[0]));
        
        for(int i=1; i < inter.length ; i++){
            if(inter[i-1][1] > inter[i][0] )
                return false ; 
        }
        return true ; 
    }
}

File: 252-meeting-rooms/252-meeting-rooms.cpp
class Solution {
public:
    bool canAttendMeetings(vector<vector<int>>& inter) {
            
        sort(inter.begin() ,inter.end() ) ;  
        for(int i=1; i < inter.size() ; i++){
            if( inter[i][0] < inter[i-1][1]) 
                return false ; 
        }
        return true ; 


    }
};